test/integration/lerna-publish.test.js

```diff
@@ -11,6 +11,23 @@ const lastCommitMessage = (cwd) =>
   execa.stdout("git", ["log", "-1", "--format=%B"], { cwd }).then(normalizeNewline);
 
 describe("lerna publish", () => {
+  test.concurrent("exists with a exit code 0 when there are no updates", async () => {
```

Comment:
They both use the same underlying utility, so the UpdatedCommand unit test is sufficient validation.